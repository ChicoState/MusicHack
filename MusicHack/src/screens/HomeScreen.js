import React from 'react';
import {Text, StyleSheet, View} from 'react-native';

const HomeScreen = () => {
  return (
    <View style={styles.viewStyle}>
      <Text style={styles.text}>Music Hack</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  text: {
    fontSize: 50,
  },
  viewStyle: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default HomeScreen;
