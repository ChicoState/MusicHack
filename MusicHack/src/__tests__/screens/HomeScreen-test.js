import React from 'react';
import HomeScreen from '../../screens/HomeScreen';

import renderer from 'react-test-renderer';

beforeAll(() => {
  HomeScreen.prototype.componentDidMount = () => {};
});

test('renders correctly', () => {
  const tree = renderer.create(<HomeScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
