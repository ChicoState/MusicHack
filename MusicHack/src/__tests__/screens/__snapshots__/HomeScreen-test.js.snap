// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 50,
      }
    }
  >
    Home Screen
  </Text>
</View>
`;
